charts/growthbook/Chart.yaml

```diff
@@ -15,22 +15,22 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.14
+version: 0.1.16
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.16.0"
+appVersion: "2.0.0"
 
 icon: https://cdn.svgporn.com/logos/growth-book-icon.svg
 
 # Disable if you already set up mongodb
-dependencies:
-  - name: mongodb
-    version: ~13.6.1
-    repository: https://charts.bitnami.com/bitnami/
-    condition: mongodb.enabled
+# dependencies:
```

Comment:
@AdamStaudt great work, I just wonder why you would remove the Mongodb dependency. I just think it should be disabled by default instead. That way it will not be installed for those who do not need it.  For those who need it, they could always override the switch

Reply:
agreed, it should be diabled by default instead, rather than removed entirely.